components/proposal-merge.tsx

```diff
@@ -80,7 +80,7 @@ export default function ProposalMerge({
   async function  getCoinInfo() { 
     await getCoinInfoByContract(activeNetwork?.networkToken?.address).then((tokenInfo) => {
       setCoinInfo(tokenInfo)
-    }).catch(err => console.error('CoinInfo', err))
+    }).catch(() => ({}));
```

Comment:
log the error out, just so we know something happened, so we can debug it in the future if something eventually breaks here